plaso/parsers/presets.py

```diff
@@ -3,75 +3,105 @@
 
 from __future__ import unicode_literals
 
+import yaml
+
+from plaso.lib import errors
+
+
 class ParserPresetDefinition(object):
```

Comment:
Please rename to ParserPreset - the definitions are the values in the Yaml file, the define the preset. Note also line 95 - "GetPresets".

Reply:
ack